src/js/components/covid19/SearchBar.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * SearchBar.jsx
+ * Created by Lizzie Salita 7/16/20
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const propTypes = {
+    setQuery: PropTypes.func.isRequired,
+    query: PropTypes.string
+};
+
+const SearchBar = ({ setQuery, query }) => {
```

Comment:
Any reason we are passing query here?

Reply:
`query` (what the table is currently filtered on) is compared to `searchString` (value of the input) to determine whether `handleClick` will reset or perform a new search (and to display the corresponding icon)

Reply:
Yes, I guess I am confused? The search bar should be taking input and searching. I do not understand why search bar needs a parent to pass it the search term? There must be a case you are trying to handle here, I do not think I see it ( possibly when a user types the same input again? ).

Reply:
Right, if their input matches the existing search it prevents submitting the same search term again and instead gives them the option to reset.

Reply:
Yeah, it just seems like this component should be self sufficient. The search bar should be able to handle that logic itself.  We could remove the query param and access the input element onSubmit to get the value or move that query parameter to the search bar like

```const SearchBar = ({ setQuery }) => {
    const [searchString, setSearchString] = useState('');
    const [currentSearchTerm, setCurrentSearchTerm] = useState('');

    const onChange = (e) => {
        setSearchString(e.target.value);
    };

    const onSubmit = (e) => {
        e.preventDefault();
        setCurrentSearchTerm(searchString);
        setQuery(searchString);
    };

    const resetSearch = (e) => {
        e.preventDefault();
        setSearchString('');
        setQuery('');
    };

    const handleClick = (e) => {
        if (searchString && currentSearchTerm === searchString) {
            resetSearch(e);
        }
        else {
            onSubmit(e);
        }
    };```

Thus, in the future when other devs implement this I need to do less work by just passing setQuery

`<SearchBar setQuery={search} />`

and not worry about handling the query param.